lib/core/src/lib/about/about-package/package-list.component.scss

```diff
@@ -14,10 +10,6 @@
     }
 
     &__row {
-        border-bottom-color: var(--adf-package-list-table-row-border-bottom-color);
-        border-style: var(--adf-package-list-table-row-border-style);
-        border-width: var(--adf-package-list-table-row-border-width);
-        border-bottom-width: var(--adf-package-list-table-row-border-bottom-width);
```

Comment:
some of those `--adf-package-list-...` variables are not used anymore, can be deleted if we don't need them anymore?